engine/src/lib.rs

```diff
@@ -498,6 +498,15 @@ mod contract {
         crate::xcc::set_wnear_address(&mut io, &Address::from_array(address));
     }
 
+    /// Returns the address for the `wNEAR` ERC-20 contract in borsh format.
+    #[no_mangle]
+    pub extern "C" fn factory_wnear_address() {
```

Comment:
I wonder if we should call it `factory_get_wnear_address` to mirror `factory_set_wnear_address`